src/disjoint_set.jl

```diff
@@ -107,17 +107,21 @@ end
 
 @compat type DisjointSets{T}
     intmap::Dict{T,Int}
+    revmap::Dict{Int,T}
```

Comment:
Couldn't this just be a `Vector{T}` of the size of the set?  That would be more efficient (in both time and space) than hashing the number.